GameOfLife/Calculations/RandomCellGenerator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+
+    public class RandomCellGenerator
+    {
+        // Generates randomly which cells are alive which not
+        public int[,] RandCells(int[,] GameField)
```

Comment:
RandCells - We try to not shorten words, where possible, as shortened words could have multiple meanings and it can start to get confusing, if you dont know the initial meaning behind the shortened word. So update to something like RandomizeCells or similar.